name.abuchen.portfolio/src/name/abuchen/portfolio/messages_de.properties

```diff
@@ -307,15 +308,21 @@ LabelSum = Summe
 
 LabelTradeCalendarDefault = Standardkalender
 
-LabelTradeCalendarEmpty = ( Keiner )
+LabelTradeCalendarEmpty = (Keiner)
+
+LabelTradeCalendarEuronext = Euronext Amsterdam
 
 LabelTradeCalendarGermany = Deutschland
 
-LabelTradeCalendarISE = Italien
+LabelTradeCalendarISE = Mail\u00E4nder B\u00F6rse
+
+LabelTradeCalendarLSE = Londoner B\u00F6rse
+
+LabelTradeCalendarNYSE = New York Stock Exchange
```

Comment:
Why “Londoner Börse” and “New York Stock Exchange”? Why not “London Stock Exchange” and “New Yorker Börse”?

Reply:
Da hast du natürlich vollkommen recht. Diese Änderung ist mir durch die Lappen gegangen.
Natürlich sollte dieser in "New York Börse" geändert werden.

https://github.com/buchen/portfolio/blob/master/name.abuchen.portfolio/src/name/abuchen/portfolio/util/TradeCalendarManager.java#L79